core/common/vitest.config.mts

```diff
@@ -1,4 +1,16 @@
 import { coverageConfigDefaults, defineConfig } from 'vitest/config';
+import * as packageJson from "./package.json";
+
+const includePackages: string[] = [];
+
+Object.keys(packageJson.peerDependencies).forEach((pkgName) => {
+  if (pkgName.startsWith("@itwin") || pkgName.startsWith("@bentley")) {
```

Comment:
i wouldn't use the name of the package... instead use the values of the package. We want any dep that has its value to be `workspace:*` to be in the includedPackages. There may be other @itwin or @bentley packages we use that aren't coming from within iTwin core